api/src/main/java/org/openmrs/api/impl/ProviderServiceImpl.java

```diff
@@ -204,6 +204,14 @@ public ProviderAttributeType getProviderAttributeType(Integer providerAttributeT
 		return dao.getProviderAttributeType(providerAttributeTypeId);
 	}
 	
+	/**
+	 * @see org.openmrs.api.ProviderService#getProviderAttributeTypeByName(java.lang.String) 
+	 */
+	@Override
+	public ProviderAttributeType getProviderAttributeTypeByName(String name) {
```

Comment:
Shouldn't this be transactional readonly?

Reply:
added!